src/styles/layout.css

```diff
@@ -12,7 +12,7 @@ body {
   /* Generic */
   --main-font-color: #1a202c;
   --main-bgd-color: #f7fafc;
-  --selection-bgd-color: #0c344b;
+  --selection-bgd-color: #a0aec0;
```

Comment:
@carlagn this will change the selection color in the whole docs repo. We need to change the color for only the dark mode code blocks.